spec/models/claim/geckoboard_event_spec.rb

```diff
@@ -17,4 +17,24 @@
       }).to have_been_requested
     end
   end
+
+  context "with lots of claims" do
+    let(:claims) { build_list(:claim, 520, :submitted, created_at: DateTime.now) }
```

Comment:
We could use `stub_const` to set `Claim::GeckoboardEvent:: BATCH_SIZE` to a smaller amount so we don't need to build so many objects here. It might speed up the spec a little bit... I have mixed feelings about stubbing constants so I'll leave it up to you.

Reply:
Good call, I don't think there's a huge performance benefit from doing this though, given that we're only building the claims, not creating them. I think I'd rather live with the slight extra fraction of a second than polluting our tests with `stub_const`